src/components/Modal.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Modal = ({ show, handleClose, children }) => {
+  const showHideClassName = show ? 'modal-wrapper display-block' : 'modal-wrapper display-none';
```

Comment:
Is this a flag for whether to show the modal or not? Could it be named something clearer? (like showModal?)